### PR TITLE
Ember bootstrap accessibility focused release

### PR DESCRIPTION
**Pull request by jaredgalanis**

### CODE REVIEW

File: source/2019-11-15-the-ember-times-issue-124.md [hunk 1]
```diff
@@ -95,13 +96,16 @@ Do you have any feedback? Feel free to reach out to [Mehul Kar (@mehulkar)](http
 
 ---
 
-## [Section Title in Title Case 🐹](#section-url)
+## [Accessibility-focused Ember Bootstrap Release ✨](https://twitter.com/simonihmig/status/1190740590377472001)
```
Comment (ijlee2) [inline source/2019-11-15-the-ember-times-issue-124.md]:
```suggestion
## [Accessibility-Focused Ember Bootstrap Release ✨](https://twitter.com/simonihmig/status/1190740590377472001)
```

File: source/2019-11-15-the-ember-times-issue-124.md [hunk 2]
```diff
@@ -95,13 +96,16 @@ Do you have any feedback? Feel free to reach out to [Mehul Kar (@mehulkar)](http
...
 
-<change section title emoji>
-<consider adding some bold to your paragraph>
+A few weeks ago [@simonihmig](https://github.com/simonihmig) and the folks at [kaliber5](https://github.com/kaliber5) released version 3.1.0 of the fantastic Ember addon [ember-bootstrap](https://github.com/kaliber5/ember-bootstrap).
```
Comment (ijlee2) [inline source/2019-11-15-the-ember-times-issue-124.md]:
```suggestion
A few weeks ago, [@simonihmig](https://github.com/simonihmig) and the folks at [kaliber5](https://github.com/kaliber5) released version 3.1.0 of the fantastic Ember addon [ember-bootstrap](https://github.com/kaliber5/ember-bootstrap). 
```

File: source/2019-11-15-the-ember-times-issue-124.md [hunk 3]
```diff
@@ -95,13 +96,16 @@ Do you have any feedback? Feel free to reach out to [Mehul Kar (@mehulkar)](http
...
 ---
 
-## [Section Title in Title Case 🐹](#section-url)
+## [Accessibility-focused Ember Bootstrap Release ✨](https://twitter.com/simonihmig/status/1190740590377472001)
 
-<change section title emoji>
-<consider adding some bold to your paragraph>
+A few weeks ago [@simonihmig](https://github.com/simonihmig) and the folks at [kaliber5](https://github.com/kaliber5) released version 3.1.0 of the fantastic Ember addon [ember-bootstrap](https://github.com/kaliber5/ember-bootstrap). 
+
+This version focuses on **improving accessibility** related concerns such as using [ember-focus-trap](https://github.com/josemarluedke/ember-focus-trap) to implement focus trap inside of modals and keyboard navigational control of drop downs. The updates also add [ember a11y tests](https://github.com/ember-a11y/ember-a11y-testing) to the test suite! 🔥🔥🔥
```
Comment (ijlee2) [inline source/2019-11-15-the-ember-times-issue-124.md]: I made these changes for brevity. What do you think?
```suggestion
This version focuses on **improving accessibility** by using [ember-focus-trap](https://github.com/josemarluedke/ember-focus-trap) to implement focus trap for modals and keyboard navigation of dropdowns. It also adds [ember a11y tests](https://github.com/ember-a11y/ember-a11y-testing) to the test suite! 🔥🔥🔥
```

File: source/2019-11-15-the-ember-times-issue-124.md [hunk 4]
```diff
@@ -95,13 +96,16 @@ Do you have any feedback? Feel free to reach out to [Mehul Kar (@mehulkar)](http
...
+
+Many thanks to all those that contributed to this release and the accessiblity concerns addressed by it 😄, including [@simonihmig](https://github.com/simonihmig), [@techn1x](https://github.com/Techn1x), [@rwachtler](https://github.com/rwachtler) and [@jelhan](https://github.com/jelhan). 
+
+You can find the release notes [here](https://github.com/kaliber5/ember-bootstrap/blob/master/CHANGELOG.md#310-2019-11-02).
```
Comment (ijlee2) [inline source/2019-11-15-the-ember-times-issue-124.md]:
```suggestion
You can find the release notes on [GitHub](https://github.com/kaliber5/ember-bootstrap/blob/master/CHANGELOG.md#310-2019-11-02).
```

**Review by ijlee2 — APPROVED** [review]
Thanks for the writeup, Jared. The updates to the addon sound amazing and I'm glad that you covered it for this week's issue!

**Comment by amyrlam** [comment]
reads great! FYI doing new format like this `[Simon Ihmig @simonihmig](https://github.com/simonihmig)` ... i need to update whats-new-in-emberland to pull the same format for the contributors corner, on the list

**Pull request merged**